app/utils/report/test.py

```diff
@@ -0,0 +1,149 @@
+# This program is free software: you can redistribute it and/or modify
```

Comment:
We should also have a `utils.report.tests.test_test_report` unit test to cover this new module (and one for bisections as well I guess).

Reply:
Also, does this pass pep8 and existing unit tests?